Cargo.toml

```diff
@@ -324,7 +324,7 @@ members = [
 panic = "unwind"
 opt-level = 3
 # AHM: FAIL-CI
-debug-assertions = true
+# debug-assertions = true # TODO
```

Comment:
@muharem there are two debug asserts failing in the new snapshot.  

One in balances underflow on releasing a hold and one in `Referenda metadata is not empty`.